docs/proposals/YYYYMMDD-template.md

```diff
@@ -122,13 +129,73 @@ place your `.plantuml` files under `images/` and run `make diagrams` from the do
 
 #### Story 2
 
+### Requirements (Optional)
+
+Some authors may wish to use requirements in addition to user stories.
+Technical requirements should derived from user stories, and provide a trace from
+use case to design, implementation and test case. Requirements can be prioritised
+using the MoSCoW (MUST, SHOULD, COULD, WON'T) criteria.
+
+The FR and NFR notation is intended to be used as cross-references across a CAEP.
+
+#### Functional Requirements
+
+Functional requirements are the properties that this design should or not include.
+
+##### FR1
+
+##### FR2
+
+#### Non-Functional Requirements
+
+Non-functional requirements are user expectations of the solution. Include
+considerations for performance, reliability and security.
+
+##### NFR1
+
+##### NFR2
+
 ### Implementation Details/Notes/Constraints
 
 - What are some important details that didn't come across above.
 - What are the caveats to the implementation?
 - Go in to as much detail as necessary here.
 - Talk about core concepts and how they releate.
 
+### Security Model
+
+Document the intended security model for the proposal, including implications
+on the Kubernetes RBAC model.
+
+#### Roles
+
+Document security roles or user personas and how they may interact with elements
+of the proposal.
+
+### RBAC
+
+RBAC (role-based access control) is the standard used for Kubernetes
+authorization. This allows users to configure who can perform actions on
+resources in specific scopes. Describe how the roles defined above will
+have read or write permissions to your proposal.
+
+
+##### Read Permissions
+
+|        | CRD 1 | CRD 2 |
+| ------ | ----- | ----- |
+| Role 1 | Yes   | Yes   |
+| Role 2 | Yes   | Yes   |
+| Role 3 | No    | Yes   |
+
+##### Write Permissions
+
+|        | CRD 1 | CRD 2 |
+| ------ | ----- | ----- |
+| Role 1 | Yes   | Yes   |
+| Role 2 | Yes   | Yes   |
+| Role 3 | No    | Yes   |
```

Comment:
Let's remove this part? Often times permissions are changed during implementation and the proposal gets outdated quickly, the most important part is above, where we require the author(s) to think about the security model and its implications. What do you think?

Reply:
Done